packages/api/tests/models/granules/test-granules-model.js

```diff
@@ -391,6 +391,23 @@ test('get() will correctly return a granule file stored using the new schema', a
   );
 });
 
+test('getRecord() returns a granule record from the database', async (t) => {
+  const granule = fakeGranuleFactoryV2();
+
+  await awsServices.dynamodbDocClient().put({
```

Comment:
Curious: why not `granule.create` here?

Reply:
When I first wrote these tests, I wanted to make sure that matching errors in `create` and `get` didn't result in tests passing that shouldn't. I don't have a particularly strong opinion on it now, but I was matching the way that the other tests in the file are written.